includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -129,13 +129,150 @@ private static function get_payment_method_configuration_from_stripe() {
 			// The API returns data for the corresponding mode of the api keys used, so we'll get either test or live PMCs, but never both.
 			if ( $configuration->parent && ( self::LIVE_MODE_CONFIGURATION_PARENT_ID === $configuration->parent || self::TEST_MODE_CONFIGURATION_PARENT_ID === $configuration->parent ) ) {
 				self::set_payment_method_configuration_cache( $configuration );
+				delete_option( 'woocommerce_stripe_pmc_fallback_id' );
 				return $configuration;
 			}
 		}
 
-		// If we don't have a Payment Method Configuration that inherits from the WooCommerce Platform, disable Payment Method Configuration sync.
-		self::disable_payment_method_configuration_sync();
-		return null;
+		WC_Stripe_Logger::warning( 'Did not find Payment Method Configuration that inherits from the WooCommerce platform' );
+
+		[
+			'pmc'    => $fallback_pmc,
+			'reason' => $fallback_reason,
+		] = self::get_fallback_payment_method_configuration( $configurations );
+
+		if ( null === $fallback_pmc ) {
+			// If we can't find a usable Payment Method Configuration, disable Payment Method Configuration sync.
+			WC_Stripe_Logger::error(
+				'No usable Payment Method Configuration found; disabling Payment Method Configuration sync',
+				[
+					'reason' => $fallback_reason,
+				]
+			);
+			self::disable_payment_method_configuration_sync();
+			return null;
+		}
+
+		WC_Stripe_Logger::debug(
+			'Using fallback Payment Method Configuration',
+			[
+				'pmc_id'   => $fallback_pmc->id,
+				'reason'   => $fallback_reason,
+				'name'     => $fallback_pmc->name ?? null,
+				'livemode' => $fallback_pmc->livemode ?? null,
+			]
+		);
+
+		self::set_payment_method_configuration_cache( $fallback_pmc );
+		return $fallback_pmc;
+	}
+
+	/**
+	 * Given the list of payment method configurations returned from Stripe,
+	 * find the fallback Payment Method Configuration to use.
+	 *
+	 * @param array $payment_method_configurations The list of payment method configurations returned from Stripe.
+	 * @return array {
+	 *     @type object|null $pmc    The fallback Payment Method Configuration.
+	 *     @type string      $reason The reason for using the fallback Payment Method Configuration.
+	 * }
+	 */
+	private static function get_fallback_payment_method_configuration( array $payment_method_configurations ): array {
+		$active_non_child_payment_method_configurations = array_filter(
+			$payment_method_configurations,
+			function ( $configuration ) {
+				if ( $configuration->parent ?? null ) {
+					return false;
+				}
+				return isset( $configuration->active ) && $configuration->active;
+			}
+		);
+
+		$fallback_pmc_id = get_option( 'woocommerce_stripe_pmc_fallback_id' );
+		$fallback_pmc    = null;
+
+		if ( [] === $active_non_child_payment_method_configurations ) {
+			if ( $fallback_pmc_id ) {
+				WC_Stripe_Logger::debug(
+					'No Payment Method Configurations returned from Stripe; deleting fallback ID option',
```

Comment:
nit: This could be somewhat misleading when we see it in logs with no other context.

```suggestion
					'No eligible Payment Method Configurations returned from Stripe; deleting fallback ID option',
```

Reply:
Good call. Fixed in 24a7dcbac26ca20df4866b8e1a6b95c6f4e2be60.